src/lib/styles/mixins/_overlay.scss

```diff
@@ -0,0 +1,5 @@
+@mixin colors {
```

Comment:
why the name `overlay` for this mixin?

Reply:
because it defines the background colors of the toast and modal

Reply:
aahh, ok, and they are overlays. Understood.